### PR TITLE
Update CI files and Badge on README

### PR DESCRIPTION
**Pull request by evillgenius75**
## Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [X ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Update CI to conform to Helium Standard and add a Badge to the Readme for CI Validation
## Issues Closed or Referenced

- Closes retaildevcrews/helium#209

### CODE REVIEW

File: .github/workflows/dockerByPR.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+name: DockerCI-PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+    paths-ignore:
+      - 'docs/**'
+      - '.github/**'
```
Comment (aflinchb) [inline .github/workflows/dockerByPR.yml]: Should update this from 'docs/**' to 'slides/**' and 'data/**'.

Comment (aflinchb) [reply .github/workflows/dockerByPR.yml]: Same in the other new yml file

Comment (evillgenius75) [reply .github/workflows/dockerByPR.yml]: Do we not want to rebuild the container if the data changes?

Comment (evillgenius75) [reply .github/workflows/dockerByPR.yml]: Fixed with commit 3e9a4b1

**Pull request merged**